### PR TITLE
Ncsdk 6532 links to subpages

### PR DESCRIPTION
**Pull request by ru-fu**
The links to subpages should look consistent.

-   Add a "Subpages:" heading to the existing docs.
-   Clean up pages where the subpages are not at the end of the page.
-   Change the list bullet to an arrow.

image

Corresponding nrfxlib PR: https://github.com/nrfconnect/sdk-nrfxlib/pull/279

### CODE REVIEW

File: doc/nrf/ug_thread_commissioning.rst [hunk 1]
```diff
@@ -209,6 +202,182 @@ See the following pages in the `OpenThread CLI Reference`_ on GitHub for an over
 * `Commissioner CLI commands`_
 * `Joiner CLI commands`_
```
Comment (greg-fer) [inline doc/nrf/ug_thread_commissioning.rst:204]: Please move this whole section, including the separator, to the very end of the file.

File: doc/nrf/libraries/nfc/lib_nfc_ndef.rst [hunk 2]
```diff
@@ -15,17 +15,17 @@ If you want to implement your own polling device, you can use the supplied parse
 
 Currently, the following NDEF libraries are supported:
```
Comment (greg-fer) [inline doc/nrf/libraries/nfc/lib_nfc_ndef.rst:16]: This sentence will not be needed with the planned updates, I think.

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: doc/nrf/ug_thread_commissioning.rst (hunk 1 above)
Comment (ru-fu) [reply doc/nrf/ug_thread_commissioning.rst]: fixed

File: doc/nrf/libraries/nfc/lib_nfc_ndef.rst (hunk 2 above)
Comment (ru-fu) [reply doc/nrf/libraries/nfc/lib_nfc_ndef.rst]: fixed

File: include/bluetooth/mesh/sensor.rst [hunk 3]
```diff
@@ -153,12 +153,7 @@ Typically, settings should only be meta data related to the sensor data type, bu
 Available sensor types
 **********************
 
-All available sensor types are collected in a single module:
-
-.. toctree::
-   :maxdepth: 1
-
-   sensor_types.rst
```
Comment (trond-snekvik) [inline include/bluetooth/mesh/sensor.rst:161]: This file is no longer part of the toctree now. Can it be re-added somewhere, like here or as a last resort, in mesh.rst?

**Review by greg-fer — APPROVED** [review]

File: doc/nrf/templates/mesh_model_main_README.rst [hunk 4]
```diff
@@ -16,16 +16,17 @@ The XYZ Mesh models allow to remotely control a disco light with a mesh device.
 
 The following XYZ models are supported:
 
+The XYZ Mesh models also feature their own common types, listed below.
+For types common to all models, see :ref:`bt_mesh_models`.
+
 .. toctree::
    :maxdepth: 1
    :glob:
+   :caption: Subpages:
```
[the remark this answers is not in the document]
Comment (ru-fu) [reply doc/nrf/templates/mesh_model_main_README.rst:25]: Right, forgot to move the introduction sentence here - thanks!

And yes, the change is for consistency. But I think it rather stresses that there are relevant subpages than remove information:
image
image

But we could of course add the list with links to the models in addition to the Subpages section, if you think it's needed?

File: include/bluetooth/mesh/sensor.rst (hunk 3 above)
Comment (ru-fu) [reply include/bluetooth/mesh/sensor.rst:161]: It's already added to mesh.rst :)

**Review by MarekPorwisz — APPROVED** [review]

**Review by pdunaj — APPROVED** [review]

**Review by b-gent — APPROVED** [review]

**Review by joerchan — APPROVED** [review]

**Pull request merged**